tools/serverpod_cli/bin/generator/class_analyzer.dart

```diff
@@ -658,4 +668,20 @@ class ClassAnalyzer {
 
     return true;
   }
+
+  bool _validateAndParseServerSideOnly(
+      YamlMap documentContents, CodeAnalysisCollector collector) {
+    var serverSideOnlyNode = documentContents.nodes['serverSideOnly'];
+    if (serverSideOnlyNode != null) {
+      if (serverSideOnlyNode.value is! bool) {
+        collector.addError(SourceSpanException(
+          'The "serverSideOnly" property must be a Boolean.',
```

Comment:
"must be a bool", to avoid any confusion.

Reply:
https://github.com/serverpod/serverpod/pull/625/commits/0fd38dfcf54bfd025936c2ab0baabc24ea1c96fb